### ISSUE TITLE
Confusing warning related to liveness probes

### ISSUE THREAD

**Issue by tengqm** [kubernetes/website#43709]
**This is a Bug Report**

**Problem:**

Well, this might be and might not be a bug. In bee671b57957fde9b97a9c4030856fc8036ffc36,
we introduced a warning to configuring readiness probe. It was something like this:

```
{{< warning >}}
Liveness probes *do not* wait for readiness probes to succeed. If you want to wait before
executing a liveness probe you should use initialDelaySeconds or a startupProbe.
{{< /warning >}}
```

I'm doubting if this is correct. We need to first ensure that a container is alive (liveness),
after which we may consider whether it is ready to receive traffic (ready).
There cannot be a container which is ready but not alive. This is intuitive.
However, we need this clarification because although we know the different problems
targeted by these two types of probes, we cannot guarantee which one reports back first.
In my understanding, the readiness probe starts ticking at the same time the liveness
probe starts working. In other words, the readiness probe doesn't wait for the liveness
probe to report okay and then start counting. That may be confusing to some users,
and that is why we need the clarification.

**Proposed Solution:**

However, the current text reads incorrect. We should instead say 
"the **readiness** probe does not wait for the **liveness** probe to succeed.".
and "If you want to wait before executing a "readiness" probe, you should use
`initialDelaySeconds` or a `startupProbe`."

**Page to Update:**

https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/

**Comment by AmarNathChary** [comment]
I would like to work on it
/assign

**Comment by sftim** [comment]
/sig node
/language en

Triage welcome!

**Comment by SergeyKanzhelev** [comment]
readiness and liveness probes are independant. We do not sequence them in any ways. So the note is correct

**Comment by AmarNathChary** [comment]
Raised PR: #43827

**Comment by sftim** [comment]
It sounds like the note _is_ confusing, even if it also correct.
/triage accepted

**Comment by AmarNathChary** [comment]
The issue was fixed by merging PR: #43827
We can close the issue
